nextcloudappstore/core/models.py

```diff
@@ -106,6 +106,7 @@ class Meta:
         verbose_name = _('App Release')
         verbose_name_plural = _('App Releases')
         unique_together = (('app', 'version'),)
+        ordering = ['-version']
```

Comment:
I'd take care of the import transforms if that's ok for you. Can you adjust the models?